action.yml

```
@@ -17,6 +17,10 @@ inputs:
     required: false
     description: The value which will be set in the project_field
     type: string
+  labels:
+    required: false
+    description: array of labels required to execute the action
```

```suggestion
    description: comma-separated list of labels required to execute the action
```

Or something like that, to make it clear how to input the array.

---

Ahh right. I think it would be nice to mention it (`See Readme for input format`) or something like that.